projects/ngx-prx-styleguide/src/lib/tz-datepicker/tz-datepicker.component.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import momentNs from 'moment'
+import 'moment-timezone/moment-timezone';
+import { TzDate } from './tzdate';
+import { map, share } from 'rxjs/operators';
+import { TzDataService } from './tz-data.service';
+
+const moment = momentNs;
+
+@Component({
+  selector: 'prx-tz-datepicker',
+  template: `
+  <ng-container *ngIf="this.model">
+    <prx-datepicker name="date"
+      [date]="this.model.pickerDate" (dateChange)="handleDateChange($event)" [changed]="changed">
+    </prx-datepicker>
+    <ng-container *ngIf="!supportsTimeInput">
+      <input (change)="handleChange()" [(ngModel)]="model.time" name="time" placeholder="08:00:00" #timestamp="ngModel" prxTzTimestamp="hh:mm:ss">
```

Comment:
I've landed on de-sugaring the two-way binding and then calling `handleChange()` from the `ngModelChange` event. We could do something super-fancy instead, like setting up an observable in the model that emits an event whenever a relevant setter is called and subscribing to that stream. Up for trying that if you think it would be a better approach.